core/src/env/engine/off_chain/test_api.rs

```diff
@@ -312,6 +331,22 @@ where
     f(default_accounts)
 }
 
+/// Runs the given closure test function with the default configuration
+/// for the off-chain environment.
+///
+/// Doesn't reuse an off-chain environment which might already exist in
+/// this thread, but instead uses a new off-chain environment instance.
+pub fn run_multiple_tests_in_thread<T, F>(f: F) -> Result<()>
```

Comment:
I currently am failing to see how this function has anything to do with running a closure in multiple threads.